scripts/script-free.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
We are committing the script for QA as well ?

Reply:
we will create another file named script-qa.sh